tests/core/tests/test_fields.py

```diff
@@ -110,6 +110,14 @@ def testget_dehydrate_method_with_custom_method_name(self):
         method_name = field.get_dehydrate_method(resource_field_name)
         self.assertEqual(method_name, custom_dehydrate_method)
 
+    def testget_dehydrate_method_with_callable(self):
```

Comment:
should the test be named `test_*` for consistency?

Reply:
yes my bad fixed in [60de1ef](https://github.com/django-import-export/django-import-export/pull/1950/commits/60de1efd887853ffc02b7c7b79c7ca2e44a7991f)